extensions/openwsman/processors/SourceInitiatedSubscriptionListener.cpp

```diff
@@ -245,8 +245,8 @@ bool SourceInitiatedSubscriptionListener::Handler::handlePost(CivetServer* /*ser
     int xml_buf_size = 0;
     ws_xml_dump_memory_node_tree_enc(node, &xml_buf, &xml_buf_size, "UTF-8");
     if (xml_buf != nullptr) {
-        core::logging::LOG_TRACE(processor_.logger_) << "Received request: \"" << std::string(xml_buf, xml_buf_size) << "\"";
-        ws_xml_free_memory(xml_buf);
+      processor_.logger_->log_trace("Received request: \"{}\"", std::string_view(xml_buf, xml_buf_size));
+      ws_xml_free_memory(xml_buf);
```

Comment:
Not strictly related to the change, but I'd move the cleanup up and out, using `gsl::finally`, just to cover the hypothetical case where the logging throws and the memory is leaked.

Reply:
good idea, fixed it in https://github.com/apache/nifi-minifi-cpp/pull/1670/commits/56cc5ac017383cb88a71caf800d46840b0a6e800
(wx_xml_free_memory does the nullptr check inside)